src/sql/command.py

```diff
@@ -120,7 +121,45 @@ def return_result_var(self):
         return self.parsed["return_result_var"]
 
     def _var_expand(self, sql, user_ns):
-        return Template(sql).render(user_ns)
+        template_render = Template(sql).render(user_ns)
+        template_render = self.remove_leading_parentheses(template_render)
+        return template_render
+
+    def remove_leading_parentheses(self, rendered_sql_command):
+        """
+        Removes any leading parentheses from the sql_command_string
```

Comment:
Looks like it removes both leading and trailing parentheses?

Reply:
@neelasha23 I've renamed it to `strip_excess_parentheses` as that made more sense as to what it accomplished.